cmd/earthly/subcmd/satellite_cmds.go

```diff
@@ -612,21 +622,31 @@ func (a *Satellite) actionRemove(cliCtx *cli.Context) error {
 		return err
 	}
 
-	found := false
+	var sat *cloud.SatelliteInstance
 	for _, s := range satellites {
 		if a.cli.Flags().SatelliteName == s.Name {
-			found = true
+			sat = &s
 			if s.Hidden {
 				return errors.New("cannot delete hidden satellites")
 			}
 		}
 	}
-	if !found {
+	if sat == nil {
 		return fmt.Errorf("could not find %q for deletion", a.cli.Flags().SatelliteName)
 	}
 
+	isOffline := sat.State == cloud.SatelliteStatusOperational || sat.State == cloud.SatelliteStatusOffline
+	if !a.force && !isOffline {
```

Comment:
Would `isOnline` be more clear?

Reply:
Yes, thanks, for noticing. I refactored and got the name backwards.

Reply:
There was more wrong than just a name. Good thing you noticed.